<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="enviar1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>211</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Presione para enviar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enviar2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>211</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Presione para enviar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enviar3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>211</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Presione para enviar</string>
   </property>
  </widget>
  <widget class="QLabel" name="recibido">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans [GOOG]</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nada en Rx</string>
   </property>
  </widget>
  <widget class="QLabel" name="diag">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans [GOOG]</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>88</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
